.github/workflows/build.yml

```diff
@@ -69,6 +69,10 @@ jobs:
       - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
         with:
           path: libtelio
+      - uses: Swatinem/rust-cache@b8a6852b4f997182bdea832df3f9e153038b5191 # v2.6.0
+        with:
+          key: ${{ matrix.target_os }}-${{ matrix.arch }}-${{ matrix.target_abi }}
+          workspaces: "libtelio -> target"
```

Comment:
nit: The documentation for this action says that target is the default, so maybe you could remove that part?